iron-collapse.html

```diff
@@ -142,7 +143,7 @@
     },
 
     updateSize: function(size, animated) {
-      this.enableTransition(animated);
+      this._updateTransition(animated);
```

Comment:
Should `animated` maybe be a legit public property, and not a private one? That way it can be set correctly by the user too.